database/seeders/TemplateSeeder.php

```diff
@@ -29,12 +164,13 @@ public function run()
             'template' => '{{$name}}',
```

Comment:
@FredPeal lets add the template for account creation, invite and the the one we have already configured